liblangutil/ErrorReporter.h

```diff
@@ -87,6 +87,34 @@ class ErrorReporter
 
 	void typeError(SourceLocation const& _location, std::string const& _description);
 
+	template <typename... Strings>
+	void typeErrors(SourceLocation const& _location, Strings const&... _descriptions)
+	{
+		std::initializer_list<std::string> const _descs = {_descriptions...};
+		std::string errorStr;
+
+		if (_descs.size() == 0)
```

Comment:
This sounds a little dangerous. I think it would be better to either assert or add the error with empty description.